blueocean-core-js/src/js/services/DefaultSSEHandler.js

```diff
@@ -59,26 +56,18 @@ export class DefaultSSEHandler {
         ];
     }
 
-    updateJob(event, overrideQueuedState) {
+    updateJob(event) {
         // const queueId = event.job_run_queueId;
         // const queueSelf = `${event.blueocean_job_rest_url}queue/${queueId}/`;
-        const runSelf = `${event.blueocean_job_rest_url}runs/${event.jenkins_object_id}/`;
-
-        for (const key of this.branchPagerKeys(event)) {
-            const pager = this.pagerService.getPager({ key });
-            this.activityService.fetchActivity(runSelf, { overrideQueuedState }).then(d => {
-                if (pager && !pager.has(runSelf)) {
-                    pager.insert(runSelf);
-                }
-                this.pipelineService.updateLatestRun(d);
-            });
-        }
```

Comment:
This code here seems responsible for adding a new run to the pager which holds runs for that job. Can it be safely removed?